versioned_docs/version-2.6/getting-started/installation-and-upgrade/install-upgrade-on-a-kubernetes-cluster/air-gapped-upgrades.md

```diff
@@ -12,9 +12,9 @@ These instructions assume you have already followed the instructions for a Kuber
 
 :::
 
-### Rancher Helm Template Options
+### Rancher Helm Upgrade Options
 
-Render the Rancher template using the same chosen options that were used when installing Rancher. Use the reference table below to replace each placeholder. Rancher needs to be configured to use the private registry in order to provision any Rancher launched Kubernetes clusters or Rancher tools.
+Upgrade with Helm using the same chosen options that were used when installing Rancher. Use the reference table below to replace each placeholder. Rancher needs to be configured to use the private registry in order to provision any Rancher launched Kubernetes clusters or Rancher tools.
```

Comment:
Make sure to mirror edits from the latest version to the versioned docs, when applicable.

Reply:
I made the edits you suggested on all three versions: 2.6, 2.7 and current. :+1: